broker/broker_grpc_pb.js

```diff
@@ -8,6 +8,7 @@
 var grpc = require('grpc');
 var ttn_broker_broker_pb = require('../broker/broker_pb.js');
 var google_protobuf_empty_pb = require('google-protobuf/google/protobuf/empty_pb.js');
+var  = require('../../../../github.com/gogo/protobuf/gogoproto/gogo_pb.js');
```

Comment:
We removed those, because they aren't used in JS.

Reply:
This is generated code, not sure what you want to be fixed here.

Reply:
This is not valid JavaScript. Sed used to delete the entire line. Now you only delete the identifier.